### PR TITLE
Add Blake

### PR DESCRIPTION
**Pull request by bcleveng**
Added My Bio

### CODE REVIEW

**Comment by MrWoodards** [comment]
Great

File: bcleveng.md [hunk 1]
```diff
@@ -0,0 +1,6 @@
+# Blake Clevenger
+
+I do Technology and stuff. I randomly decided to start making candles... because I need more hobbies.
```
Comment (dherbert1) [inline bcleveng.md:3]: Favorite scent so far?

Comment (rttowne) [inline bcleveng.md:3]: Interesting hobby.

Comment (kgeers3) [reply bcleveng.md:3]: Ooh candles sounds fun!

Comment (bcleveng) [reply bcleveng.md:3]: Coffee is pretty solid but there have been some surprises like "Tobacco and Bayleaf" is pretty amazing.

**Pull request merged**